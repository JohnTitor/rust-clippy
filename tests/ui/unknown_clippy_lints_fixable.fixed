// run-rustfix

#![allow(clippy::all)]
#![warn(clippy::pedantic)]

// check if suggesting similar lint
#[warn(clippy::match_as_ref)]
#[warn(clippy::if_not_else)]
#[warn(clippy::inherent_to_string_shadow_display)]
// check if suggesting similar lowercase lint
#[warn(clippy::unit_cmp)]
fn main() {}
