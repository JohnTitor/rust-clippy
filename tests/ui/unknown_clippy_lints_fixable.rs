// run-rustfix

#![allow(clippy::All)]
#![warn(clippy::pedantic)]

// check if suggesting similar lint
#[warn(clippy::match_sa_ruf)]
#[warn(clippy::if_not_els)]
#[warn(clippy::inherent_to_strin_shadew_display)]
// check if suggesting similar lowercase lint
#[warn(clippy::Unit_Cm)]
fn main() {}
