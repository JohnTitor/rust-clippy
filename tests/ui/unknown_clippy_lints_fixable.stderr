error: unknown clippy lint: clippy::match_sa_ruf
  --> $DIR/unknown_clippy_lints_fixable.rs:7:8
   |
LL | #[warn(clippy::match_sa_ruf)]
   |        ^^^^^^^^^^^^^^^^^^^^ help: did you mean: `clippy::match_as_ref`
   |
   = note: `-D clippy::unknown-clippy-lints` implied by `-D warnings`

error: unknown clippy lint: clippy::if_not_els
  --> $DIR/unknown_clippy_lints_fixable.rs:8:8
   |
LL | #[warn(clippy::if_not_els)]
   |        ^^^^^^^^^^^^^^^^^^ help: did you mean: `clippy::if_not_else`

error: unknown clippy lint: clippy::inherent_to_strin_shadew_display
  --> $DIR/unknown_clippy_lints_fixable.rs:9:8
   |
LL | #[warn(clippy::inherent_to_strin_shadew_display)]
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: did you mean: `clippy::inherent_to_string_shadow_display`

error: unknown clippy lint: clippy::Unit_Cm
  --> $DIR/unknown_clippy_lints_fixable.rs:11:8
   |
LL | #[warn(clippy::Unit_Cm)]
   |        ^^^^^^^^^^^^^^^ help: did you mean: `clippy::unit_cmp`

error: unknown clippy lint: clippy::All
  --> $DIR/unknown_clippy_lints_fixable.rs:3:10
   |
LL | #![allow(clippy::All)]
   |          ^^^^^^^^^^^ help: lowercase the lint name: `clippy::all`

error: aborting due to 5 previous errors

